design-tokens/tokens/layer1/base.json

```diff
@@ -113,78 +125,150 @@
       "type": "letterSpacing"
     },
     "increased": {
-      "value": "150%",
+      "value": "2%",
       "type": "letterSpacing"
     },
     "decreased": {
-      "value": "-5%",
+      "value": "-2%",
       "type": "letterSpacing"
     }
   },
   "paragraphSpacing": {
-    "h1": {
-      "value": "32",
+    "small": {
+      "value": "1rem",
+      "type": "paragraphSpacing"
+    },
+    "medium": {
+      "value": "1.125rem",
       "type": "paragraphSpacing"
     },
-    "h2": {
-      "value": "26",
+    "large": {
+      "value": "1.375rem",
       "type": "paragraphSpacing"
     }
   },
   "fontWeights": {
-    "headingRegular": {
-      "value": "Regular",
-      "type": "fontWeights"
-    },
-    "headingBold": {
+    "bold": {
       "value": "Bold",
       "type": "fontWeights"
     },
-    "bodyRegular": {
-      "value": "Regular",
+    "semiBold": {
+      "value": "Semi Bold",
       "type": "fontWeights"
     },
-    "bodyBold": {
-      "value": "Bold",
+    "regular": {
+      "value": "Regular",
       "type": "fontWeights"
     }
   },
   "fontSizes": {
-    "h1": {
-      "value": "{fontSizes.h2} * 1.25",
+    "headingXXXL": {
+      "value": "2.125rem",
       "type": "fontSizes"
     },
-    "h2": {
-      "value": "{fontSizes.h3} * 1.25",
+    "headingXXL": {
+      "value": "1.5rem",
       "type": "fontSizes"
     },
-    "h3": {
-      "value": "{fontSizes.h4} * 1.25",
+    "headingXL": {
+      "value": "1.25rem",
       "type": "fontSizes"
     },
-    "h4": {
-      "value": "{fontSizes.h5} * 1.25",
+    "headingL": {
+      "value": "1rem",
       "type": "fontSizes"
     },
-    "h5": {
-      "value": "{fontSizes.h6} * 1.25",
+    "headingM": {
+      "value": "0.85rem",
       "type": "fontSizes"
     },
-    "h6": {
-      "value": "{fontSizes.body} * 1",
+    "headingS": {
+      "value": "0.75rem",
       "type": "fontSizes"
     },
-    "body": {
-      "value": "16",
+    "bodyLarge": {
+      "value": "1rem",
       "type": "fontSizes"
     },
-    "sm": {
-      "value": "{fontSizes.body} * 0.85",
+    "bodyMedium": {
+      "value": "0.875rem",
       "type": "fontSizes"
     },
-    "xs": {
-      "value": "{fontSizes.body} * 0.65",
+    "bodySmall": {
+      "value": "0.75rem",
       "type": "fontSizes"
     }
```

Comment:
We shouldn't have a 0.75rem, it's 12px it's too small, we dropped 12px font size everywhere with the MIUI effort we shouldn't go back to it.
Smallest should be 14px IMO.

Same applies fro headingS by the way.

Reply:
Discussed and reviewed at the office with @gregaubert. We are making the conscious decision to keep 12px which should only be used when displaying meta-data. Documentation to support this design decision will follow so we can ensure that this value is not used wrongfully like it was in the past.